azure_functions_worker/dispatcher.py

```diff
@@ -570,7 +570,9 @@ def tp_max_workers_validator(value: str) -> bool:
             if int_value < PYTHON_THREADPOOL_THREAD_COUNT_MIN or (
                     int_value > PYTHON_THREADPOOL_THREAD_COUNT_MAX):
```

Comment:
I still don't understand why we even have this check at all. Regardless of the MAX value, what is the reason?

As a user, I expect the value I specify to be either accepted (preferred) or at least explicitly rejected (if there is an actual reason). Right now, if I specify 1025, this value will not be accepted and, even worse, the worker will fall back to the very low default, and I may not notice the warning. And, _in order to achieve these troubles_, we have quite a lot of code!

If we didn't have this check at all, what would be the motivation for us to add it? Are we assuming we know more than the user who specified a certain value? How can we know that?

Reply:
After offline conversation - we have decided to remove any artificial limits on the maximum number of threads.